backend/config/corpora-api.yml

```diff
@@ -329,6 +329,10 @@ paths:
                 properties:
                   dataset_uuid:
                     $ref: "#/components/schemas/dataset_uuid"
+                  processing_status:
+                    type: string
+                    enum:
+                      [ PENDING, SUCCESS,FAILURE]
```

Comment:
nit: uneven spacing! e.g., `[ PENDING, SUCCESS,FAILURE]` to `[PENDING, SUCCESS, FAILURE]`

I'll add a commit to run `prettier` on yaml 👌